eppo_client/models.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+from typing import Any, Dict, List, Optional
+
+from eppo_client.base_model import SdkBaseModel
+from eppo_client.rules import Rule
+
+
+class Variation(SdkBaseModel):
+    key: str
+    value: str | int | float | bool
```

Comment:
We also have JSON-typed variations. Should add `Dict[str, Any]`, unless that's too general and we want to restrict the type of the JSON values.

Reply:
I figured a json blob is a string

Reply:
This sounds nice -- I hadn't thought about typed values at all so far